src/board/system76/common/peci.c

```diff
@@ -29,8 +29,9 @@ static uint8_t FAN_HEATUP[BOARD_HEATUP] = { 0 };
 
 static uint8_t FAN_COOLDOWN[BOARD_COOLDOWN] = { 0 };
 
-// Tjunction = 100C for i7-8565U (and probably the same for all WHL-U)
+#ifndef T_JUNCTION
 #define T_JUNCTION 100
```

Comment:
Why not use PECI and read the appropriate MSR to get TJunction? It is available in 
TEMPERATURE_TARGET_MSR (0x1a2) bits 23:16

Reply:
I just tried it. Trying to read MSR 0x1a2 yields PECI return code 0x90 - invalid / illegal request. Intel doc #630138 has a table (3-10) of MSRs accessible through RdIAMSR, 0x1a2 is not on that list

Reply:
Hmm, might still be readable via RdPkgConfig, will try it out

Reply:
@miczyg1 done in https://github.com/Dasharo/ec/pull/69/commits/48dad9b3ac853ff651ca9c0bd112a5e6f686235f